attribute-projections/src/main/java/org/hypertrace/core/attribute/service/projection/StringConcatenation.java

```diff
@@ -0,0 +1,35 @@
+package org.hypertrace.core.attribute.service.projection;
+
+import java.util.List;
+import org.hypertrace.core.attribute.service.v1.AttributeKind;
+
+public class StringConcatenation implements AttributeProjection {
+  public static StringConcatenation STRING_CONCATENATION = new StringConcatenation();
+
+  public static String concatenate(List<String> strings) {
```

Comment:
Allow for a delimiter to be passed in?

Reply:
Is there a delimiter?

Reply:
Concatenate doesn't generally take a delimiter - join does. So three options:
1. Rename operator to join and add delimiter, which could be provided as (or potentially default to) empty
1. Two separate operators, join and concat with join requiring a delimiter
1. Leave it as is, because join is just syntax sugar on concat, and I can already define the args to this as any mix of literals/attributes/other projections.

What do you think?

Reply:
2. Clean and separate.